jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -93,21 +93,19 @@ private Tracer(
     this.zipkinSharedRpcSpan = zipkinSharedRpcSpan;
     this.activeSpanSource = activeSpanSource;
 
-    int ip;
-    try {
-      ip = Utils.ipToInt(Inet4Address.getLocalHost().getHostAddress());
-    } catch (UnknownHostException e) {
-      ip = 0;
-    }
-    this.ip = ip;
-
     this.version = loadVersion();
 
     tags.put("jaeger.version", this.version);
     String hostname = getHostName();
     if (hostname != null) {
       tags.put("jaeger.hostname", hostname);
     }
+    int ipv4 = 0;
+    try {
+      tags.put("ip", InetAddress.getLocalHost().getHostAddress());
```

Comment:
We define tracer tags at initialization and never update them. If IP address changes, there's nothing we can do without refactoring. I don't think it's a big concerns.